### PR TITLE
modify docs for great renaming

### PR DESCRIPTION
**Pull request by djrtwo**
Attempted to be thorough in specs and READMEs.

Did not:
* rename eth2spec package
* rename any fields related to eth1 data voting

### CODE REVIEW

File: specs/altair/fork.md [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 ## Introduction
 
-This document describes the process of the first upgrade of Ethereum 2.0: the Altair hard fork, introducing light client support and other improvements.
+This document describes the process of the first upgrade of the eacon chain: the Altair hard fork, introducing light client support and other improvements.
```
Comment (protolambda) [inline specs/altair/fork.md:20]:
```suggestion
This document describes the process of the first upgrade of the beacon chain: the Altair hard fork, introducing light client support and other improvements.
```

Comment (djrtwo) [reply specs/altair/fork.md:20]: surprised this wasn't caught by spell check. looking to see if altair/ is being checked

File: specs/altair/validator.md [hunk 2]
```diff
@@ -49,8 +49,8 @@ This is an accompanying document to [Ethereum 2.0 Altair -- The Beacon Chain](./
 
 ## Introduction
 
-This document represents the expected behavior of an "honest validator" with respect to the Altair upgrade of the Ethereum 2.0 protocol.
-It builds on the [previous document for the behavior of an "honest validator" from Phase 0](../phase0/validator.md) of the Ethereum 2.0 protocol.
+This document represents the expected behavior of an "honest validator" with respect to the Altair upgrade of the Ethereum consensus protocol.
+It builds on the [previous document for the behavior of an "honest validator" from Phase 0](../phase0/validator.md) of the Ethereum consensus protocol.
```
Comment (hwwhww) [inline specs/altair/validator.md:53]: Ah, I just saw you used `Ethereum proof-of-stake protocol` to replaced them on phase0 spec. Either way is fine with me.

```suggestion
This document represents the expected behavior of an "honest validator" with respect to the Altair upgrade of the Ethereum proof-of-stake protocol.
It builds on the [previous document for the behavior of an "honest validator" from Phase 0](../phase0/validator.md) of the Ethereum proof-of-stake protocol.
```

**Review by hwwhww — APPROVED** [review]
I added some commits to address some missing sections.
LGTM 👍

File: specs/altair/validator.md (hunk 2 above)
Comment (djrtwo) [reply specs/altair/validator.md]: modded back to pos for consistency

**Pull request merged**